project-base/storefront/components/Blocks/Product/ButtonsAction/ProductComparePopup.tsx

```diff
@@ -13,15 +13,15 @@ type ProductComparePopupProps = {
 export const ProductComparePopup: FC<ProductComparePopupProps> = ({ onCloseCallback }) => {
     const t = useTypedTranslationFunction();
     const { url } = useDomainConfig();
-    const [productsComparisonUrl] = getInternationalizedStaticUrls(['/products-comparison'], url);
+    const [productComparisonUrl] = getInternationalizedStaticUrls(['/product-comparison'], url);
```

Comment:
To me it seems like it is comparing products not single product, so I am wondering why it needed to be changed? 🤔

Reply:
I would refer to Cambridge Dictionary:
https://dictionary.cambridge.org/dictionary/english/product-comparison

Furthermore, I apply the same logic as with other groups of items, such as:

- product list - list of products
- product category - category of products
etc.